src/utils/format.ts

```diff
@@ -127,7 +129,9 @@ export const prettyNumbers = (x: string) => {
 };
 
 export const formatAddress = (address: string) => {
-  return address.substring(0, 6) + '...' + address.substring(address.length - 4, address.length);
+  return isValidAddress(address)
+    ? address.substring(0, 6) + '...' + address.substring(address.length - 4, address.length)
```

Comment:
can u use etherscans getAddress so we format to checksum too please

Reply:
fixed